docs/conf/portland/2019/cfp.rst

```diff
@@ -24,46 +23,78 @@ Between the talks on stage, the discussions in the unconference, the collaborati
+We would much rather hear about process, culture, data, people, or the metaphysical side effects of spending your life thinking about docs.
 
 **Audience awareness**
 
-When crafting talk proposals, remember that you're going to be talking to a mix of levels of expertise, skill sets, and professions. Your talk doesn't have to be relevant to everyone, but it should be relevant to most people and shouldn't
-make too many assumptions about what people already know.
+When crafting talk proposals, remember that you're going to be talking to a mix of levels of expertise, skill sets, and professions.
+Your talk doesn't have to be relevant to everyone, but it should be relevant to most people and shouldn't make too many assumptions about what people already know.
+If you are making those assumptions about what your audience knows, it helps everyone if you state them up front explicitly.
 
-Take a look at the abstracts for accepted talks from the last couple of conferences for some ideas:
+It can be  helpful to check out topics that might be related to your talk from previous years as well:
 
+* `Portland 2018 <http://www.writethedocs.org/conf/portland/2018/speakers/>`_
+* `Prague 2018 <http://www.writethedocs.org/conf/prague/2018/speakers/>`_
 * `Portland 2017 <http://www.writethedocs.org/conf/na/2017/speakers/>`_
 * `Prague 2017 <http://www.writethedocs.org/conf/eu/2017/speakers/>`_
-* `Portland 2016 <http://www.writethedocs.org/conf/na/2016/speakers/>`_
-* `Prague 2016 <http://www.writethedocs.org/conf/eu/2016/speakers/>`_
 
-Need help?
------------
 
-If you need a hand preparing your talk proposal, there are lots of good places to start:
+Who we're looking for
+---------------------
+
+The short description here is "if you care about documentation and have something to say about it, we want you to speak"!
+That said, there are some more helpful and concrete pointers below.
+
+**Diverse group of speakers**
+
+We welcome talks from first-time speakers, from industry experts, and from everyone in between.
+Whatever your background and experience, we prefer hearing about new approaches than about tried-and-tested technology.
+We especially welcome talks from underrepresented groups within the tech community.
+
+**Mix of roles and perspectives**
+
+Talks from the scientific community, fiction writers, system administrators, and support staff – in addition to technical writers and software developers – are all valuable to our attendees.
+
+**Diverse audience**
+
+Likewise, we love the diversity of our community, and we encourage support-folks to attend talks about programming, developers to attend talks about writing, writers to attend talks about data, and so on and so forth.  
+
+Interested in speaking but don't have a topic?
+-----------------------------------------------
+
+If you need a hand preparing or honing your talk proposal, there are lots of good places to start:
```

Comment:
Maybe we should focus a bit on hesitance or things like that too?

How about (language will need tweaking):

==================

Hesitant to submit, or don't have a topic?
-----------------------------------------

As we said above, we welcome a wide variety of topics, backgrounds and experience level. We don't think badly of you if we aren't able to accept your proposal, so if you're not sure whether your topic is a good fit, feel you don't have enough speaking experience for a conference, or you think someone else may be able to give a better talk on your topic, we strongly encourage you to submit regardless.

If you need a hand in preparing or honing your talk proposal, there are lots of good places to start: